docs/migrate-tags.md

```diff
@@ -0,0 +1,169 @@
+# `migrate-tags`
```

Comment:
These docs will be helpful for migrating the `design-tokens` and `eslint-plugin-design-tokens` packages, and they also reference a tool to assist with the migrations. Once those packages are migrated, we can decide whether to keep this documentation or remove it, as we likely won’t have any further packages to migrate, unlike core.

Reply:
Create a ticket to decide whether to remove migration scripts and documentation once we have migrated design-tokens